ports/nrf/README.md

```diff
@@ -10,50 +10,41 @@ This is a port of MicroPython to the Nordic Semiconductor nRF series of chips.
 * Pins
 * ADC
 * I2C
-* PWM (nRF52 only)
+* PWM
 * Temperature
 * RTC (Real Time Counter. Low-Power counter)
 * BLE support including:
-  * Peripheral role on nrf51 targets
-  * Central role and Peripheral role on nrf52 targets
+  * Peripheral role
+  * Scanner role
   * _REPL over Bluetooth LE_ (optionally using WebBluetooth)
-  * ubluepy: Bluetooth LE module for MicroPython
+  * ubluepy: Bluetooth LE module for CircuitPython
   * 1 non-connectable advertiser while in connection
 
 ## Tested Hardware
 
-* nRF51
-  * [micro:bit](http://microbit.org/)
-  * PCA10000 (dongle)
-  * PCA10001
-  * PCA10028
-  * PCA10031 (dongle)
-  * [WT51822-S4AT](http://www.wireless-tag.com/wireless_module/BLE/WT51822-S4AT.html)
 * nRF52832
   * [PCA10040](http://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf52%2Fdita%2Fnrf52%2Fdevelopment%2Fnrf52_dev_kit.html) 
   * [Adafruit Feather nRF52](https://www.adafruit.com/product/3406)
-  * [Thingy:52](http://www.nordicsemi.com/eng/Products/Nordic-Thingy-52)
-  * [Arduino Primo](http://www.arduino.org/products/boards/arduino-primo)
 * nRF52840
   * [PCA10056](http://www.nordicsemi.com/eng/Products/nRF52840-Preview-DK)
 
 ## Compile and Flash
 
 Prerequisite steps for building the nrf port:
 
-    git clone <URL>.git micropython
-    cd micropython
+    git clone <URL>.git circuitpython
+    cd circuitpython
     git submodule update --init
     make -C mpy-cross
 
-By default, the PCA10040 (nrf52832) is used as compile target. To build and flash issue the following command inside the ports/nrf/ folder:
+By default, the PCA10040 (nRF52832) is used as compile target. To build and flash issue the following command inside the ports/nrf/ folder:
```

Comment:
The default is actually `feather52` now, with S132v2.0.1 and BLE enabled by default.